dev/com.ibm.ws.security.acme/resources/com/ibm/ws/security/acme/resources/AcmeMessages.nlsprops

```diff
@@ -274,23 +275,50 @@ CWPKI2059I=CWPKI2059I: Certificate revocation status checking found that the cer
 CWPKI2063E.explanation=The certificate authority updated its terms of service and requires user interaction.
 CWPKI2063E.useraction=Review the provided terms of service.
 
 CWPKI2064I=CWPKI2064I: The ACME service retrieved the certificate with the {0} serial number from the {1} URI in {2} seconds.
 CWPKI2064I.explanation=The ACME service successfully requested a certificate.
-CWPKI2064I.useraction=No action is required.
\ No newline at end of file
+CWPKI2064I.useraction=No action is required.
+
+CWPKI2065W=CWPKI2065W: The ACME service failed to automatically renew the certificate with the {0} serial number. The request is scheduled to try again in {1}. The certificate expires on {2}. The renew request error is ''{3}''.
+CWPKI2065W.explanation=The ACME service tried to renew a certificate but encountered an error. The ACME service continues to request a new certificate until a new certificate is issued.
+CWPKI2065W.useraction=Review the error message for details on the failure.
+
+CWPKI2066E=CWPKI2066E: The ACME service failed to automatically renew the certificate with the {0} serial number. The certificate is revoked. The request is scheduled to try again in {1}. The renew request error is ''{2}''.
+CWPKI2066E.explanation=The ACME service tried to renew a certificate but encountered an error. The ACME service continues to request a new certificate until a new certificate is issued. SSL/TLS requests fail until a new certificate request is successful.
+CWPKI2066E.useraction=Review the error message for details on the failure. 
+
+CWPKI2067I=CWPKI2067I: The certificate with the {0} serial number is revoked. The ACME service requests a new certificate from the ACME certificate authority at the {1} URI.
+CWPKI2067I.explanation=When the ACME service detects that the certificate is revoked, it automatically requests a new certificate.
+CWPKI2067I.useraction=No action is required.
+
+CWPKI2068W=CWPKI2068W: The ACME service automatic certificate checking failed to check if the certificate with the {0} serial number is expiring or revoked. The check is scheduled to try again in {1}. The error is ''{2}''.
+CWPKI2068W.explanation=The ACME service started checking if the certificate is expiring or revoked, but failed.
+CWPKI2068W.useraction=Review the error message for details on the failure. Review the certificate status using the ACME REST interface. If the certificate needs to be renewed, use the ACME REST interface to request a new certificate.
+
+CWPKI2069I=CWPKI2069I: The ACME service automatic certificate checking is disabled. Expiring or revoked certificates are not automatically renewed.
+CWPKI2069I.explanation=The ACME service does not check for expiring or revoked certificates on an automated schedule. If the certificate expires or is revoked, TLS/SSL requests cannot complete unless the server is restarted or the REST interface is used to renew the certificate.
```

Comment:
Change `SSL/TLS requests` to `SSL and TLS requests`

Reply:
Changed